rpcs3/Emu/Cell/PPULLVMRecompiler.h

```diff
@@ -8,12 +8,12 @@
 #include "Emu/Cell/PPUDecoder.h"
 #include "Emu/Cell/PPUThread.h"
 #include "Emu/Cell/PPUInterpreter.h"
-#include "llvm/ExecutionEngine/ExecutionEngine.h"

```

Comment:
If it was like this before doesn't mean it's a good thing. What if in some feature llvm release MCJIT.h will stop including ExecutionEngine.h? I don't know how hardly it depends on it so that's why I'm not sure.